README.md

```diff
@@ -1099,6 +1099,9 @@ _See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managemen
 - [Gitea](https://gitea.io) - Community managed fork of Gogs, lightweight code hosting solution. ([Demo](https://try.gitea.io), [Source Code](https://github.com/go-gitea/gitea)) `MIT` `Go`
 - [GitLab](http://gitlab.org/) - Self Hosted Git repository management, code reviews, issue tracking, activity feeds and wikis. ([Demo](https://gitlab.com/), [Source Code](https://gitlab.com/gitlab-org/gitlab-ce)) `MIT` `Ruby`
 - [Gitlist](http://gitlist.org/) - Web-based git repository browser - GitList allows you to browse repositories using your favorite browser, viewing files under different revisions, commit history and diffs. ([Source Code](https://github.com/klaussilveira/gitlist)) `BSD-3-Clause` `PHP`
+- [Gitolite](http://gitolite.com/gitolite/index.html) - Gitolite allows you to
+  setup git hosting on a central server, with fine-grained access control and
+many more powerful features. ([Source Code](https://github.com/sitaramc/gitolite)) `GPLv2` `Perl`
```

Comment:
* Please put the new entry on a single line (not 3 lines)
 * Please use GPL-2.0 license code
 * Is it actively maintained?

Reply:
It is actively maintained.